cdap-app-templates/cdap-etl/hydrator-spark-core-base/src/main/java/io/cdap/cdap/etl/spark/batch/ETLSpark.java

```diff
@@ -95,6 +95,9 @@ public void initialize() throws Exception {
 
     SparkConf sparkConf = new SparkConf();
     sparkConf.set("spark.speculation", "false");
+    // turn off auto-broadcast by default until we better understand the implications and can set this to a
+    // value that we are confident is safe.
+    sparkConf.set("spark.sql.autoBroadcastJoinThreshold", "-1");
```

Comment:
Is there any constant class we can use for these config?

Reply:
not that I know of